projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -184,6 +184,10 @@ public function woocommerce_is_active() {
 		return false;
 	}
 
+	public function editing_toolkit_is_active() {
+		return true;
```

Comment:
I was wondering about the default return value here; you mention how similar to `woocommerce_is_active` this is. In this case, shouldn't we default to `false` as well?